go/cmd/vtctld/vtctld.go

```diff
@@ -321,5 +321,21 @@ func main() {
 		http.Redirect(w, r, target, http.StatusFound)
 	})
 
+	// serve some data
+	knownCellsCache := newKnownCellsCache(ts)
+	http.HandleFunc("/json/KnownCells", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Ideally, we need to unify all our json responses. Other urls have ?format=json. Which way should we go?
The other issue to discuss is it should be a REST API, although I still don't know what that exactly means :).